modern/_tests/workingdir/simple/test_simple.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
I wanted to move all as Python scripts, including those test_xxx.sh files. When I started this new feature, it was all bash script, but when I opened the PR, I converted some of them to Python. WDYT, should we convert all to Python?

Reply:
I would say that we need them as `.sh` script. With _bash_ we can run them in _all_ the images, there is no need to install python. For example, we could run these tests in `builder` images.